lib/helpers/calculator_helpers.js

```diff
@@ -18,41 +18,36 @@ module.exports = class CalculatorHelpers {
    *   their key is 'value' and value is an array of results. Second result is the the episode results keyed by
    *   episode id and with the value being a set just like the patient results.
    */
-  static createPopulationValues(measure, population, results, patientId, observationDefs, populationIndex) {
+  static createPopulationValues(measure, populationSet, patientResults, observationDefs) {
```

Comment:
With `population` being changed to `populationSet`, we have everything we need to build `observationDefs` within this function rather than passing it in as a parameter.  Currently, with all observations being the same, this isn't needed, but `observationDefs` is locked to `measure.population_sets[0]` in calculator.js, so if they are different in the future, it will need to be moved into this function instead.

Reply:
If this observationDef generation was added to this function, and we generated the observationsDefs for each population_set, then we would be adding duplicate copies of the generated elm onto the mainLibraryElm. The observations in measure.population_sets[0] are the same in every single population_set

`mainLibraryElm.library.statements.def.push(generatedELMJSON);`